### ISSUE TITLE
Draw Unrevealed issues

### ISSUE THREAD

**Issue by jasonconlon** [GeckoTH/arkham-horror#216]
In the 'Carnevale of Horrors' scenario, there is a set of 7 double-sided cards, which start in the Special deck.  On their back-side is the 'Masked Carnevale Goer' image, and the cards must be dealt out on to the table with this back-side showing.
If you choose "Draw Unrevealed" to attempt to do this, there's a glitch:  The cards appear with their standard back (which is either Encounter back or Player Card back, depending on what their front-side card template is), instead of the 'Masked Carnevale Goer' image; and if you try to "Flip" the card, even the front-side now shoes with the standard back as well instead of the actual front-side image.
If you simply perform a regular "Draw", then everything works normally, with the correct front-side image, and when you "Flip" the card it also has the correct 'Masked Carnevale Goer' image on the back-side.  Unfortunately this impacts the experience if you do a regular "Draw", because it loads the image front-side first and gives away information that should remain hidden.

**Comment by KenDdoox** [comment]
It will be fixed when the new PR is merged. Please go to the module's website to download the updated image pack for the scenario and install it after the new update (1.7.6.7) is live.

**Comment by jasonconlon** [comment]
Thanks a bunch!

**Comment by jasonconlon** [comment]
I've just tested it, and sorry to report that now it's broken in a different way.  The back-side image should show the 'Masked Carnevale-Goer' image, but it no longer does.  When I "Draw" the card from the Special deck, it still shows the front-face image, but when I "Flip" the card now the other side also shows the front-face image - like it's double-sided with the same image on both sides.

Also just FYI, the original issue around using the 'Draw Unrevealed' feature is still present, as described in the original post above.

**Comment by KenDdoox** [comment]
Did you download the updated image pack for the scenario? You must have the correct images for it to work.

As for the "Draw Unrevealed" issue, it's because these cards are double sided, which interacts weirdly with how that function works. We'll look into it later.

**Comment by jasonconlon** [comment]
That's weird.  It's working now!  Yay!  Thanks heaps.

Just FYI -
Initially, I started and updated OCTGN, downloaded the new image pack for Carnevale of Horrors, loaded in the image pack, and then started a new game with the Carnevale of Horrors deck - and was seeing that issue,
I've just restarted OCTGN, started a new game with the Carnevale of Horrors deck, and now the images are as they should be.

One other minor glitch I did notice, though.  When you first start up a game (and load up a game deck such as Carnevale of Horrors), the Ctrl-Shift-E function doesn't work until you perform some other action first, and then it works fine.  Strangely, the problem isn't there when you invoke a "Reset" and load up the game deck; it's only after the initial game start.
